src/nyc_trees/apps/core/models.py

```diff
@@ -88,6 +95,10 @@ def clean(self):
     def get_absolute_url(self):
         return reverse('user_detail', args=[self.username])
 
+    def completed_training(self):
```

Comment:
As long as a Parks admin can mark someone as trained without the person actually clicking through the process, then we are covered.